db/migrate/20171211124439_add_uuid_index_to_katello_subscription_facets.rb

```diff
@@ -0,0 +1,5 @@
+class AddUuidIndexToKatelloSubscriptionFacet < ActiveRecord::Migration
```

Comment:
The class name doesn't match the filename  (Facet vs Facets), causing the tests to fail.

Reply:
oops, fixed.